usr/include/pkcs11types.h

```diff
@@ -950,6 +980,11 @@ typedef CK_ULONG CK_MECHANISM_TYPE;
 #define CKM_ECDSA_SHA256               0x00001044
 #define CKM_ECDSA_SHA384               0x00001045
 #define CKM_ECDSA_SHA512               0x00001046
+/* The following are new for v3.0 */
+#define CKM_ECDSA_SHA3_224             0x00001047
+#define CKM_ECDSA_SHA3_256             0x00001048
+#define CKM_ECDSA_SHA3_384             0x00001049
+#define CKM_ECDSA_SHA3_512             0x0000104a
```

Comment:
Hm, above you used a capital letter 'A' in the hex value 0x0000039A ... maybe you want to make this consistent

Reply:
fixed